stories/index.js

```diff
@@ -66,11 +66,19 @@ stories.add('all icons', () => {
           </div>
         );
       }
+  ));
+
+stories.add('Colored icon', () => {
+  return (
+    <h2 style={ { 'color': 'red' } }>
+      Test Me Red <Icon name='bell' size='large' />
+    </h2>
   );
+});
 
 stories.add('large icon', () => {
   return (
-    <Icon name='bell' size='large' />
+    <Icon name='bell' size='large' fill={ text('color', 'hotpink') } />
```

Comment:
Did you mention you want to remove this `hotpink` color? I'm good either way!

Reply:
Good catch, I removed it from the all icons story, but kept is here because it's a good override test. Once we have regression testing around this basically all stories we write turn into automatic UI tests—I am so jazzed about this!